OrbitGl/App.cpp

```diff
@@ -772,8 +772,16 @@ void OrbitApp::AddUiMessageCallback(
   m_UiCallback = a_Callback;
 }
 
-//-----------------------------------------------------------------------------
+bool OrbitApp::IsCapturing() {
```

Comment:
`Capture::IsCapturing()` does exactly the same, should we keep only one?

Reply:
done.